examples/BUILD

```diff
@@ -22,9 +22,14 @@ cc_fuzz_test(
     srcs = ["buffer_overflow_fuzz_test.cc"],
 )
 
+# This target shows how to create a fuzz test target with corpus files
```

Comment:
Retrofitting an existing target for corpus support creates confusion (and the fact that we need a comment to clarify it is another sign of it IMO).

How about creating a new cc_fuzz_test() target called "empty_fuzz_test_with_corpus" that re-uses the "empty_fuzz_test.cc" source file, but adds corpus support?